### ISSUE TITLE
Darkmoon Faire quests not recorded

### ISSUE THREAD

**Issue by SpareSimian** [SavedInstances/SavedInstances#307]
I've relied in past DMFs on SI to tell me which alts have completed DMF each month. This month (September 2019) I'm not seeing them reported in the weekly list. There were some glitches at Blizzard for this month's DMF. First, it wasn't listed in the calendar, but they fixed that. Then it started a week late, on the 2nd Sunday of the month. Could those be affecting the listing?

**Comment by SpareSimian** [comment]
Yep, it's the late DMF that caused it. Change line 737 in SavedInstances.lua to add a week to the end-day calculation:

    dmf_end.day = firstsunday + 14 -- 1 days of "slop" and allow for one week delay (September 2019 as example)

**Comment by Stanzilla** [comment]
I'm not sure if we should account for what is probably a one time fail here

**Comment by SpareSimian** [comment]
Is there anything that would be bad about extending the end time? In fact, why not extend it to the end of the month?

**Comment by LiangYuxuan** [comment]
Extending the end time would let Darkmoon Faire's quests keep being shown in whole month, and might break the monthly reset on these quest.

**Comment by dblanch369** [comment]
Stan, I wish I could say that it's a one time fail, but it actually happened earlier this year as well, with no explanation from Blizzard for the week bump.  Maybe it's better to be safe than sorry?

**Comment by SpareSimian** [comment]
OTOH, Blizzard has started DMF a day or two early when the 1st falls on a Sunday, so it starts a couple days before the end of the previous month. I think November was such a case, as I didn't check and missed it then. I don't mind it showing all month, but it should probably cancel around the 21st.
